### PR TITLE
klaviyo - minor fixes

### PR DESCRIPTION
**Pull request by utsabc**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/215)

### CODE REVIEW

File: integrations/Klaviyo/browser.js [hunk 1]
```diff
@@ -57,20 +57,26 @@ class Klaviyo {
       return;
     }
     let payload = {
-      $id: message.userId || message.anonymousId,
+      $id: message.userId || message.context.userId || message.anonymousId,
```
Comment (sayan-mitra) [inline integrations/Klaviyo/browser.js:60]: let's use get for this since we now have the lib?

File: integrations/Klaviyo/browser.js [hunk 2]
```diff
@@ -57,20 +57,26 @@ class Klaviyo {
...
       $email: message.context.traits.email,
       $phone_number: message.context.traits.phone,
-      $first_name: message.context.traits.firstName,
-      $last_name: message.context.traits.lastName,
+      $first_name:
```
Comment (sayan-mitra) [inline integrations/Klaviyo/browser.js:63]: same as above

File: integrations/Klaviyo/browser.js [hunk 3]
```diff
@@ -57,20 +57,26 @@ class Klaviyo {
...
+        message.context.traits.firstName ||
+        message.context.traits.firstname ||
+        message.context.traits.first_name,
+      $last_name:
```
Comment (sayan-mitra) [inline integrations/Klaviyo/browser.js:67]: same as above

File: integrations/Klaviyo/browser.js (hunk 1 above)
Comment (utsabc) [reply integrations/Klaviyo/browser.js:60]: yes

File: integrations/Klaviyo/browser.js (hunk 3 above)
Comment (sayan-mitra) [reply integrations/Klaviyo/browser.js:67]: also let's move these logic to utils for now?
something like getDefinedTraits(message) ==> return {email, phone, lastName, firstName, ..etc}

**Review by sayan-mitra — APPROVED** [review]

**Pull request merged**